### ISSUE TITLE
why val_loss is getting higher and higher？

### ISSUE THREAD

**Issue by zbsean** [matterport/Mask_RCNN#100]
why val_loss is getting higher and higher？	
How to choose the optimal model

**Comment by waleedka** [comment]
Since you haven't provided any details, the only thing I could suggest is lowering the learning rate and trying again.
